SecurityPkg/SecurityPkg.dsc

```diff
@@ -71,6 +71,8 @@
   TcgStorageCoreLib|SecurityPkg/Library/TcgStorageCoreLib/TcgStorageCoreLib.inf
   TcgStorageOpalLib|SecurityPkg/Library/TcgStorageOpalLib/TcgStorageOpalLib.inf
   ResetSystemLib|MdeModulePkg/Library/BaseResetSystemLibNull/BaseResetSystemLibNull.inf
+  VariableKeyLib|SecurityPkg/Library/VariableKeyLibNull/VariableKeyLibNull.inf
+  RpmcLib|SecurityPkg/Library/RpmcLibNull/RpmcLibNull.inf

```

Comment:
RpmcLib is not related to the other changes in the commit.

Reply:
Honestly not sure - they're included in the commit and I havent dropped them yet because I'm not sure why they were added

Reply:
They were used in a POC for platform input to support replay protection and encryption of UEFI variables.